frontend/src/index.jsx

```diff
@@ -20,4 +21,6 @@ const App = () => {
   );
 };
 
-ReactDOM.render(<App />, document.getElementById('root'));
+const AppWithStore = createStore(App);
```

Comment:
Noted; obsolete since I will change the design to your suggestion [here](https://github.com/ibeckermayer/teleport-interview/pull/3#discussion_r555537683)